src/main/java/org/apache/sling/feature/launcher/impl/Main.java

```diff
@@ -66,9 +67,11 @@
 
     public static final String OPT_REPOSITORY_URLS = "u";
 
-    public static final String OPT_CONFIG_CLASH = "CC";
+    public static final String OPT_CONFIG_CLASH = "C";
 
-    public static final String OPT_ARTICACT_CLASH = "C";
```

Comment:
This seems wrong - I don't think we want to change any of the switches at this point. We could consider adding new ones that are essentially aliases for the old switches but we have to stay backwards compatible at this point. 

Furthermore, I think you don't reflect this change in the help nor in the env var mappings.

Reply:
but that means that you are not able to use the options properly.

Example: you want to set the ConfigClash `a=1` then you use `-CCa=1`.
The Parser will decide on `-C` that you mean the ARTICACT_CLASH and uses `Ca=1` as KeyVal.

```
java.lang.IllegalArgumentException: Unable to parse mvn coordinates/url: Ca
	at org.apache.sling.feature.ArtifactId.parse(ArtifactId.java:103)
	at org.apache.sling.feature.launcher.impl.Main.lambda$parseArgs$2(Main.java:257)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.apache.sling.feature.launcher.impl.Main.lambda$parseArgs$3(Main.java:257)
	at java.base/java.util.Optional.ifPresent(Optional.java:176)
	at org.apache.sling.feature.launcher.impl.Main.parseArgs(Main.java:256)

```

Reply:
right, but you would right them as "-CC a=1", so the space does disambiguate them, no?

Reply:
right! i made a lot of [tests](https://github.com/apache/sling-org-apache-sling-feature-launcher/blob/3479c6ea3bc51148043a4c4d56fd0de6da2ae52b/src/test/java/org/apache/sling/feature/launcher/impl/MainTest.java#L202-L209) for all the options.

Here is the test that would fail:
https://github.com/apache/sling-org-apache-sling-feature-launcher/blob/3479c6ea3bc51148043a4c4d56fd0de6da2ae52b/src/test/java/org/apache/sling/feature/launcher/impl/MainTest.java#L202-L209

Reply:
Thanks. As I said, we could consider introducing aliases that don't have this problem and maybe removing the old switches from the printHelp and the documentation - but we have to support them for the time being as this is already used in various places.